### PR TITLE
Add Traefik v1.7

### PR DESCRIPTION
**Pull request by SantoDE**
This PR ports the [Stable Traefik Helm Chart](https://github.com/helm/charts/tree/master/stable/traefik) to the rancher catalog.

Feedback etc, welcome! :)

### CODE REVIEW

File: charts/traefik/v1.7/questions.yml [hunk 1]
```diff
@@ -0,0 +1,118 @@
+categories:
+- Proxy
+- Loadbalancer
```
Comment (guangbochen) [inline charts/traefik/v1.7/questions.yml]: can we please add the following label, will need to for our multi-cluster app filtering, thanks.
```
labels:
  io.cattle.role: project # options are cluster/project
```

Comment (SantoDE) [reply charts/traefik/v1.7/questions.yml]: Done :)

**Comment by guangbochen** [comment]
LGTM.

**Comment by SantoDE** [comment]
Thanks @guangbochen. What is now needed to get that merged? :) ping @deniseschannon maybe? :) Thanks.

**Comment by guangbochen** [comment]
merge into dev and we will sync to master later on after the QA did the test, thanks.

**Pull request merged**